Session2.Rmd

```diff
@@ -237,27 +238,37 @@ ggplot(data = cycleway_Delft) +
   coord_sf(datum = st_crs(28992))
 ```
 
```

Comment:
lower down, `ggtitle()` has been replaced with `labs()` ( also when only title and subtitle are given as arguments). Is it a conscious decision to show two ways of going about this or  should it be aligned?